### ISSUE TITLE
[CI] Athena driver test flake

### ISSUE THREAD

**Issue by nemanjaglumac** [metabase/metabase#28859]
Example of a failed run:
https://github.com/metabase/metabase/actions/runs/4312058246/jobs/7522154799

```
FAIL in metabase-enterprise.sandbox.api.user-test/update-user-attributes-test (http_client.clj:170)

With premium token features = #{"sandboxes"} 404 if user does not exist
PUT http://localhost:3001/api/mt/user/5/attributes expected a status code of 404, got 200.
expected: 404
  actual: 200


FAIL in metabase-enterprise.sandbox.api.user-test/update-user-attributes-test (user_test.clj:78)

With premium token features = #{"sandboxes"} 404 if user does not exist
expected: "Not found."
  actual: true
```

**Comment by camsaul** [comment]
Doesn't seem like an Athena-specific thing?

**Comment by metamben** [comment]
Fixed in #28886.
